pkg/types/nutanix/platform.go

```diff
@@ -0,0 +1,46 @@
+package nutanix
+
+// Platform stores any global configuration used for Nutanix platforms.
+type Platform struct {
```

Comment:
Enhancement 912 references a [container](https://github.com/openshift/enhancements/pull/912/files#diff-74fc807d5f6f39d7ce057f41e1197572515da4c728dbc70dcfc91668ed7486b9R87) that is required.  Is that still required?

Reply:
Good catch. Added it.